examples/ct/ct_server/main.go

```diff
@@ -54,48 +57,66 @@ func loadTrustedRoots() (*ct.PEMCertPool, error) {
 }
 
 func loadLogKeys() (crypto.KeyManager, error) {
+	if len(*privateKeyPEMFlag) == 0 {
+		return nil, errors.New("the --private_key_file flag must be set to reference a valid PEM file")
+	}
+	if len(*publicKeyPEMFlag) == 0 {
+		return nil, errors.New("the --public_key_file flag must be set to reference a valid PEM file")
+	}
+
 	logKeyManager := crypto.NewPEMKeyManager()
 
 	privateKeyPEM, err := ioutil.ReadFile(*privateKeyPEMFlag)
-
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to load private key file: %v", err)
 	}
 
 	err = logKeyManager.LoadPrivateKey(string(privateKeyPEM), *privateKeyPasswordFlag)
-
 	if err != nil {
-		return nil, fmt.Errorf("failed to load private key: %v", err)
+		return nil, fmt.Errorf("failed to parse private key: %v", err)
 	}
 
 	publicKeyPEM, err := ioutil.ReadFile(*publicKeyPEMFlag)
-
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to load public key file: %v", err)
 	}
 
 	err = logKeyManager.LoadPublicKey(string(publicKeyPEM))
 
 	if err != nil {
-		return nil, fmt.Errorf("failed to load public key: %v", err)
+		return nil, fmt.Errorf("failed to parse public key: %v", err)
 	}
 
 	return logKeyManager, nil
 }
 
+func awaitSignal() {
+	// Arrange notification for the standard set of signals used to terminate a server
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+
+	// Now block main and wait for a signal
+	sig := <-sigs
+	glog.Warningf("Signal received: %v", sig)
+	glog.Flush()
+
+	// Terminate the process
+	os.Exit(1)
+}
+
 func main() {
 	flag.Parse()
+	glog.CopyStandardLogTo("WARNING")
+	glog.Info("**** CT HTTP Server Starting ****")
```

Comment:
Again it's not a big issue. There isn't a CT RPC server so it's possibly not adding value. I'll leave this to you.